### ISSUE TITLE
desc and altText forms on update action should be hided as optional

### ISSUE THREAD

**Issue by silloi** [nota/n8n-nodes-gyazo#33]
#32
> In my opinion, Description and Alt Text should be optional parameters, and the values should be not empty string but undefined unless they are selected.

- Form fields for desc and altText  on update action should be hided as "Optional" parameters
- Unless they are added as optional fields by clicking "Add Fields" (like search action), their value should be undefined
- If the fields are visible, their value (empty string or not) should be registered to request body fields
